app-emulation/docker/docker-18.03.1.ebuild

```diff
@@ -282,6 +282,8 @@ src_install() {
 	doman man/man*/*
 
 	dobashcomp contrib/completion/bash/*
+	insinto /usr/share/fish/completions/
```

Comment:
please install 3rd party fish completions to `/usr/share/fish/vendor_completions.d/`
that is intended dir for packages to install completions to.
I should probably document it somewhere )